### DISCUSSION TITLE
Processing failed: This show isn't in your list, you need to add it before post-processing an episode

### DISCUSSION THREAD

**Discussion by mitch71h** [pymedusa/Medusa#11076] [Q&A]
I added the show 1923 a few weeks ago and typo'd the directory as 1932. When the 1st episode hit a few days ago, I noticed Medusa was not finding any episodes. I renamed the directory to 1923 and then added a scene exception to set the scene name to 1923. Neither change allowed Medusa to find the episode when doing a manual search or recognize a manually downloaded episode during post-processing. I then tried removing the show and chose "Delete Files." I rebooted the PC and then re-added 1923. Medusa still could not find episodes with a manual search or during post-processing. I noticed the original "1923" scene exception showed up again after re-adding the show, so I believe the old settings are surviving the remove/delete - re-adding process. Do you have any solutions for this issue or how to completely scrub the database of a show before trying to re-add it?

**Comment by nydave69** [comment]
Not sure if this is related;
But I keep loosing the show exceptions that I have entered..  And have to re-enter them weekly,  I've tried to narrow it down as to when/why they keep diapering, only thing I have come up with is when the nightly show update happens. If I manually do a show update it has no effect. This happens with all shows...   Also I can't filter on postprocessng in the log ,  The filter doesn't match the log and can't be changed.
